Gemfile.lock

```diff
@@ -0,0 +1,37 @@
+GEM
```

Comment:
The file `Gemfile.lock` should not be committed since we're developing a plugin. I've just added it into the `.gitignore` on `master`. Can you remove this file from this branch? Thanks!